lava_test_plans/testcases/android-vts.yaml

```diff
@@ -0,0 +1,12 @@
+{% extends "testcases/master/template-android-xts.jinja2" %}
+
+{% set xts_test_url = TEST_VTS_URL %}
+{% set xts_version = TEST_VTS_VERSION %}
+
+{% set xts_test_params = xts_test_params|default("vts") %}
+{% set test_name = test_name|default("vts") %}
+{% set xts_test_suite_name = xts_test_suite_name|default("vts-test") %}
+{% set xts_test_path = xts_test_path|default("android-vts") %}
+{% set xts_pkg_name = xts_pkg_name|default("android-vts.zip") %}
+{% set xts_expect_reboot = xts_expect_reboot|default(true) %}
+{% set xts_metadata_prefix = xts_metadata_prefix|default("vts") %}
```

Comment:
please move this file to lava_test_plans/testcases/master/template-android-vts.jinja2
and make it a template.